jobs/JGLOBAL_ATMOS_SFCANL

```diff
@@ -24,6 +24,9 @@ YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
 YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
   COMOUT_ATMOS_RESTART:COM_ATMOS_RESTART_TMPL
 
+RUN="enkfgdas" MEMDIR="ensstat" YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+  COMIN_ATMOS_ENKF_ANALYSIS_STAT:COM_ATMOS_ANALYSIS_TMPL
```

Comment:
This is looking for the current ensemble update analysis stat directory correct? Does having `RUN="enkgdas"` cause a failure during the gfs cycle?

Reply:
I've turned off DO_GSISOILDA for the gfs runs.